README.md

```diff
@@ -75,6 +75,48 @@ These three concepts have a specific meaning in the context of sumo_db.
 TO implement an adapter, you must implement two behaviors: `sumo_backend` and
 `sumo_store`.
 
+## Events
+
+Sumo throws events when things happen. An Event has this structure:
+```erlang
+{EventId, Model, Event, Args}
+```
+- `EventId` is a `reference()` which identifies the event
+- `Model` is the model where the event happend, for example, if we are creating a new entitiy in the model `people` the value of `Model` would be `people`.
+- `Event` is the type of the event.
+- `Args` extra data sent.
+
+Those are the current supported events:
+
+- `pre_persisted` just before persisting some entity. This event has the entity we want to persist as an `Args`. It is throw on this function:
+    - `sumo:persist/2`
+- `persisted` just after persisting some entity. This event hast the persisted entity as an `Args` It is throw on this function:
```

Comment:
Add a note stating that it keeps the same EventId as the corresponding `pre_persisted`